README.md

```diff
@@ -174,6 +174,14 @@ foreach (Form form in shipment.forms) {
 }
 ```
 
+### Testing
+
+All .NET/.NET Core versions of the library utilize a VCR tool (`Scotch`) to record and replay HTTP requests and responses via cassettes. These cassettes are used by our .NET/.NET Core unit tests; no live API calls are made.
```

Comment:
```suggestion
All .NET/.NET Core versions of the library utilize a VCR tool (`Scotch`) to record and replay HTTP requests and responses via cassettes. These cassettes are used by our .NET/.NET Core unit tests; no live API calls are made once a cassette has been recorded as the cassette gets replayed instead of the live API call.
```

Reply:
Resolved in 88726861fba93fd348a76b99a851ecbdaf1c49d5